test/test_functionalization.py

```diff
@@ -150,6 +196,20 @@ def f(x):
 
 
 
+def forward(self, a_1):
+    empty = torch.ops.aten.empty.SymInt([4], dtype = torch.float32, device = device(type='cpu'), pin_memory = False)
+    empty_1 = torch.ops.aten.empty.SymInt([4], dtype = torch.float32, device = device(type='cpu'), pin_memory = False)
```

Comment:
These are dead, but once we run reinplacing we cannot run standard DCE because it will delete too many nodes. I guess we should make a mutation sound DCE.

Reply:
Actually, this is the functional graph. This doesn't seem that interesting from a functionalization perspective lol; although it suggests we should run DCE after functionalizing.

Reply:
oh yep, agreed - and we already run DCE as part of partitioning today, so these will get removed.